README.md

```diff
@@ -15,10 +13,168 @@
+
+1. Connected components that simplify complex cloud-connected workflows, like Authenticator.
+2. Primitive components that create consistency across Amplify UI and allow you to build complete applications that fit your brand, like Buttons and Badges.
+3. Data-bound components that make it easy to display dynamic data, like DataStoreCollections.
+4. Theming capabilities that allow you to customize the appearance of Amplify UI to match your brand.
+
+**This is an early preview of the upcoming Amplify UI 2.0 release.**
+
+## Getting started
+
+We will be publishing the next major version (2.0) of these packages under the `@next` tag in npm when they are ready:
+
+- @aws-amplify/ui-react
+- @aws-amplify/ui-angular
+- @aws-amplify/ui-vue
+
+Bug fixes to the existing stable release of the ui packages will happen on the [aws-amplify/amplify-js](https://github.com/aws-amplify/amplify-js) repository.
+
+## Usage
+
+### React
+
+```js
+import { Authenticator, useAuth, Button, Heading } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css';
+
+export default function App() {
+  const { state, user, signOut } = useAuth();
+
+  if (state !== 'authenticated') {
+    return <Authenticator />;
+  }
+
+  return (
+    <>
+      <Heading level={1}>Welcome {user.username}!</Heading>
+      <Button variation="primary" onClick={signOut}>
+        Sign Out
+      </Button>
+    </>
+  );
+}
+```
+
+[Head over to the docs for more information on React usage](https://ui.docs.amplify.aws/ui/components/authenticator?platform=react)
+
+### Vue
+
+```html
+<authenticator>
+  <template v-slot="{ user }">
+    <h1>Welcome {{ user.username }}!</h1>
+    <button @click="send('SIGN_OUT')">Sign Out</button>
+  </template>
+</authenticator>
+```
+
+[Head over to the docs for more information on Vue usage](https://ui.docs.amplify.aws/ui/components/authenticator?platform=vue)
+
+> Note: primitives not available for Vue yet
+
+### Angular
+
+```html
+<amplify-authenticator>
+  <ng-template amplifyOverride="authenticated" let-username="username">
+    <h1>Welcome, {{username}}!</h1>
+    <amplify-sign-out></amplify-sign-out>
+  </ng-template>
+</amplify-authenticator>
+```
+
+[Head over to the docs for more information on Angular usage](https://ui.docs.amplify.aws/ui/components/authenticator?platform=angular)
+
+> Note: primitives not available for Angular yet
 
 ## Documentation
 
-The latest documentation is found here:
+- Docs for the latest stable version of Amplify UI can be found [here](https://docs.amplify.aws/ui).
+- Docs for the next version of Amplify UI can be found [here](https://ui.docs.amplify.aws).
+
+Improving our documentation and providing better and more interactive sample code is one of the focuses of the new Amplify UI. We also wanted to make it easier to keep our docs in sync with the UI code so now this repository will contain all the documentation, UI code, and end-to-end tests to ensure the correctness of the documentation.
```

Comment:
Maybe adding where in the repo these elements are located. For example "...all the documentation (/docs), UI code (/packages),...."